app/services/clean_unreceived_matches.rb

```diff
@@ -0,0 +1,17 @@
+class CleanUnreceivedMatches
+  def perform
+    ActiveRecord::Base.transaction do
+      Match.missing.each do |m|
+        UnreceivedMatch.create(
+          user_id: u.user_id,
+          receiver_id: u.receiver_id,
+          code: u.code,
+          received: false,
+        )
+        m.delete
```

Comment:
Use `destroy!` (notice the `!`) or this will not throw exceptions. Delete skips validations and usually doesn't throw exceptions (unless a SQL error is thrown like a foreing key error)

Reply:
:+1: